ibis/bigquery/compiler.py

```diff
@@ -586,6 +586,19 @@ def compiles_floor(t, e):
     return 'CAST(FLOOR({}) AS {})'.format(t.translate(arg), bigquery_type)
 
 
+@compiles(ops.CMSMedian)
+def compiles_approx(translator, expr):
+    arg = expr.op().args[0]
+    where = expr.op().args[1]
```

Comment:
Let's just pull out `expr.op()` into a variable and use the `arg` and `where` properties.